File: Examples/Ruby/Document/put_merge_documents.rb
require 'aspose_pdf_cloud'

class Document

  include AsposePDFCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @pdf_api = PdfApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new 
    response = @storage_api.put_create(file_name, File.open("../data/" << file_name,"r") { |io| io.read } )
  end

  def put_merge_documents
    upload_file("Sample.pdf")
    upload_file("sample-input.pdf")

    merge_documents = MergeDocuments.new
    merge_documents.list = ["Sample.pdf", "sample-input.pdf"]

    response = @pdf_api.put_merge_documents("sample-merged.pdf", merge_documents)
  end

end

document = Document.new()
puts document.put_merge_documents

File: Examples/PHP/Document/PutMergeDocuments.php
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\PDF\PdfApi;
use Aspose\PDF\AsposeApp;

class Document {

	public $pdfApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->pdfApi = new PdfApi();
	}

	public function putMergeDocuments() {
		$fileName = "sample-merged.pdf";
		// Upload file to Aspose Cloud Storage
		$fileName1 = "Sample.pdf";
		Utils::uploadFile($fileName);
		$fileName2 = "sample-input.pdf";
		Utils::uploadFile($fileName1);

		$body = array('List' => array($fileName1, $fileName2));
		$result = $this->pdfApi->PutMergeDocuments($fileName, $storage = "", $folder = "", $body);
		print_r ( $result );
	}
}

$document = new Document();
$document->putMergeDocuments();

?>